packages/flutter/lib/src/material/chip.dart

```diff
@@ -540,7 +548,8 @@ class InputChip extends StatelessWidget
   /// The [onPressed] and [onSelected] callbacks must not both be specified at
   /// the same time.
   ///
-  /// The [label], [isEnabled] and [selected] arguments must not be null.
+  /// The [label], [isEnabled], [selected], and [clipBehavior] arguments must
+  /// not be null.
```

Comment:
Looks like you need an `assert(clipBehavior != nll)`

Reply:
Done.